apps/meteor/client/views/admin/manageSubscription/components/GetMoreWithEnterprise.tsx

```diff
@@ -0,0 +1,70 @@
+import { Box, States, StatesIcon, StatesTitle, StatesSubtitle, Grid, Button } from '@rocket.chat/fuselage';
+import { Card, CardBody, CardTitle, FramedIcon } from '@rocket.chat/ui-client';
+import type { ReactElement } from 'react';
+import React, { memo } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { PRICING_LINK } from '../utils/links';
+
+const GetMoreWithEnterprise = (): ReactElement => {
+	const { t } = useTranslation();
+
+	const enterpriseSectionContent = [
```

Comment:
I would suggest using object mapping and improving the keys of the translations, using a contextualized approach. E.g: GetMoreWithEnterprise_whatIsAbout_Title

Reply:
Since this content can change I would not recommend contextualizing the keys

Reply:
The suggestion was made considering hypothesis to use it in another places, but since it's very specific we should keep with your proposal